examples/c/mass_spring_model/mass_spring_qp.c

```diff
@@ -212,7 +212,7 @@ ocp_qp_in *create_ocp_qp_in_mass_spring(void *config, ocp_qp_dims *dims)
     int *nu = dims->nu;
     int *nb = dims->nb;
     int *ng = dims->ng;
-    int *ns = dims->ns;
+    // int *ns = dims->ns;
```

Comment:
this will create a problem if one comments the soft_constraints back in, no?

Reply:
no, there is a separate function for the case with soft constraints
create_ocp_qp_in_mass_spring_soft_constr